source/extensions/access_loggers/grpc/grpc_access_log_impl.h

```diff
@@ -21,7 +21,19 @@ namespace Extensions {
 namespace AccessLoggers {
 namespace GrpcCommon {
 
-// TODO(mattklein123): Stats
+/**
+ *  * All stats for the HTTP/1 codec. @see stats_macros.h
+ *   */
+#define ALL_GRPC_ACCESS_LOGGER_STATS(COUNTER)                                                      \
```

Comment:
re: where to document stats, in a perfect world we would I think update https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log so it's a bit more hierarchical and then you could have a section on "Access log sinks" (or similar) and then document the stats within the page for the gRPC access loggers...